src/couch_index/src/couch_index_server.erl

```diff
@@ -201,50 +215,50 @@ handle_info(Msg, State) ->
 code_change(_OldVsn, State, _Extra) ->
     {ok, State}.
 
-handle_config_change("couchdb", "index_dir", RootDir, _, RootDir) ->
-    {ok, RootDir};
-handle_config_change("couchdb", "view_index_dir", RootDir, _, RootDir) ->
-    {ok, RootDir};
-handle_config_change("couchdb", "index_dir", _, _, _) ->
-    exit(whereis(couch_index_server), config_change),
+handle_config_change("couchdb", "index_dir", RootDir, _, #st{root_dir = RootDir} = St) ->
+    {ok, St};
```

Comment:
Was wondering why we have both "view_index_dir" and "index_dir". I couldn't find anything using `"index_dir", it seems odd to keep monitoring it for changes when it doesn't seem to be used anywhere?

Reply:
I think it's the old name for it (from a _long_ time ago). all I'm doing here is changing how we match RootDir, the view_index_dir / index_dir is historical and unrelated to my change.

Reply:
Ah that's fine. It should be a separate PR to clean it up.